src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -235,12 +234,9 @@ Vector2i RenderTarget::mapCoordsToPixel(const Vector2f& point, const View& view)
     const Vector2f normalized = view.getTransform().transformPoint(point);
 
     // Then convert to viewport coordinates
-    Vector2i        pixel;
     const FloatRect viewport = FloatRect(getViewport(view));
-    pixel.x                  = static_cast<int>((normalized.x + 1.f) / 2.f * viewport.width + viewport.left);
-    pixel.y                  = static_cast<int>((-normalized.y + 1.f) / 2.f * viewport.height + viewport.top);
-
-    return pixel;
+    return Vector2i(Vector2f((normalized.x + 1.f) / 2.f * viewport.width + viewport.left,
+                             (-normalized.y + 1.f) / 2.f * viewport.height + viewport.top));
```

Comment:
I want to change this to match the new version we're using in `mapPixelToCoords`